apps/files/src/components/TransferOwnershipDialogue.vue

```diff
@@ -50,10 +50,13 @@
 					@search="findUserDebounced" />
 			</p>
 			<p>
-				<input type="submit"
-					class="primary"
-					:value="submitButtonText"
+				<NcButton nativeType="submit"
+					type="primary"
 					:disabled="!canSubmit">
+					<template #default>
+						{{ submitButtonText }}
+					</template>	
```

Comment:
```suggestion
					{{ submitButtonText }}
```
Don't need to pass default template explicitly